nixos/tests/systemd.nix

```diff
@@ -145,8 +145,8 @@ import ./make-test-python.nix ({ pkgs, ... }: {
 
         output = machine.succeed("systemctl show | grep Watchdog")
         assert "RuntimeWatchdogUSec=30s" in output
```

Comment:
For some reason this value isn't actually set, but stays at `0` (the others do get set).

I'm not sure if this is an upstream bug or something only happening inside the VM - maybe someone can take a closer look.

Reply:
https://github.com/systemd/systemd/issues/15726 & https://github.com/systemd/systemd/pull/15932 may be relevant here.

Reply:
@Emantor this is only for per-service watchdog properties, which can be changed via the `sd_notify` protocol, isn't it?

Here, it's specifically the system-wide `RuntimeWatchdogUSec` property which doesn't seem to be honored, `RebootWatchdogUSec` and `KExecWatchdogUSec` are changed, which is really confusing.